### DISCUSSION TITLE
Date insertion always 0001-01-01 00:00:00

### DISCUSSION THREAD

**Discussion by obex1** [chuot/rdio-scanner#250] [Q&A] [answered]
Hi, 

I am new to rdio_scanner, and I started using it with "Tetra Trunck Tracker" which produce recordings with files in the format "[time] [unit] [group]" time is like hhmmss in 24hrs format

I am also using anther audio recorder that produce files as we wish, I am producing "[date] [time] [group] [hz]" -- date as "0001_00_00" with underscore and time as 'hh-mm-ss" with dashes.

All files get inserted fine except for the datestime stamp in the database.. is there a way to solve this? or get current time as a default timestamp in the entry? does this affect pruning? (having all files with same date time)

Thanks
Khalid A.

**Comment by chuot** [comment, marked as answer]
The #TIME and #ZTIME works only with #DATE. The time will not be taken into account if #DATE isn't used. The pruning does use the call.DateTime and will remove all calls if they're all have their DateTime set to 0001-01-01 00:00:00.

I will add something to the upcoming release which will use the current date and time if none are provided by the meta tags. It should help your case scenario.

**Comment by obex1** [comment]
I just tried v. 6.5.3  now it works perfectly for me everything is sorted out. I loved it.

THANK YOU
